Patrycja_W.txt

```diff
@@ -0,0 +1 @@
+Jestem Patrycja i chcę dzisiaj zrobić trening. 
```

Comment:
Cześć Patrycja, fajnie, że jesteś - proszę jeszcze o rozwiązanie konfliktu ;)

Reply:
Konflikt rozwiązany :)

Reply:
Prawie w porządku - jednak lepiej by było oryginalny plik zostawić, jak jest, a dostosować swoje zmiany :) (tzn. zostawić Patryka tam gdzie jest, a swoją zmianę dopisać za nim - bezpieczniejsza opcja :) ).
I w międzyczasie ktoś jeszcze wskoczył, więc masz jeszcze jedną okazję poćwiczyć :D (potem już Ciebie zmerguję jako pierwszą i inni będą mieć konflikty a nie Ty, obiecuję :D )

Reply:
Dzięki za tę uwagę :) zgadzam się i zdecydowanie ma to sens :)